src/frontend/screens/ObservationEdit/index.tsx

```diff
@@ -87,7 +85,7 @@ export const ObservationEdit: NativeNavigationComponent<'ObservationEdit'> & {
         <ThumbnailScrollView />
       </ScrollView>
       <BottomSheet items={bottomSheetItems} />
-      <ErrorModal sheetRef={sheetRef} closeSheet={closeSheet} isOpen={isOpen} />
+      <ErrorBottomSheet error={error} clearError={() => setError(null)} />
```

Comment:
I think we should only pass a meaningful clear error to the bottom sheet, such as [resettingMutationState](https://tanstack.com/query/latest/docs/framework/react/guides/mutations#resetting-mutation-state). If we can't pass a way to meaningfully clear the error, than we should just pass a go back function, which navigates the user back so they can start the process again.

Reply:
It makes sense for me know :) Thank you

Reply:
Let me know if current solution is okay :)